apps/crn-server/test/data-providers/working-groups.data-provider.test.ts

```diff
@@ -219,9 +224,15 @@ describe('Working Group Data Provider', () => {
         expect(response?.members).toStrictEqual([
           {
             user: {
-              ...getSquidexUserGraphqlResponse().findUsersContent,
-              ...getSquidexUserGraphqlResponse().findUsersContent?.flatData,
-              displayName: 'Tom Hardy',
+              id: graphqlUser.id,
```

Comment:
since almost all properties are the same can't you use spread and override the ones that are different?

Reply:
I ended up removing from the query a lot of fields that were not used in the frontend, so I think it's better to pick the fields that were selected in the query. 

if I use this expect

```
        expect(response?.members).toStrictEqual([
          {
            user: {
              ...getSquidexUserGraphqlResponse().findUsersContent,
              ...getSquidexUserGraphqlResponse().findUsersContent?.flatData,
              displayName: `${graphqlUser.flatData.firstName} ${graphqlUser.flatData.lastName}`,
            },
          },
        ]);
```

it gives me:


```
    expect(received).toStrictEqual(expected) // deep equality

    - Expected  - 165
    + Received  +   1

      Array [
        Object {
          "user": Object {
    -       "alumniLocation": null,
            "alumniSinceDate": null,
    -       "avatar": Array [],
    -       "biography": "some bio",
    -       "city": "London",
    -       "contactEmail": "T@rdy.io",
    -       "country": "United Kingdom",
    -       "created": "2020-09-23T20:45:22Z",
    -       "degree": "MPH",
    -       "dismissedGettingStarted": false,
    +       "avatarUrl": undefined,
            "displayName": "Tom Hardy",
            "email": "H@rdy.io",
    -       "expertiseAndResourceDescription": null,
    -       "expertiseAndResourceTags": Array [
    -         "expertise 1",
    -         "expertise 2",
    -         "expertise 3",
    -         "expertise 4",
    -         "expertise 5",
    -       ],
            "firstName": "Tom",
    -       "flatData": Object {
    -         "alumniLocation": null,
    -         "alumniSinceDate": null,
    -         "avatar": Array [],
    -         "biography": "some bio",
    -         "city": "London",
    -         "contactEmail": "T@rdy.io",
    -         "country": "United Kingdom",
    -         "degree": "MPH",
    -         "dismissedGettingStarted": false,
    -         "email": "H@rdy.io",
    -         "expertiseAndResourceDescription": null,
    -         "expertiseAndResourceTags": Array [
    -           "expertise 1",
    -           "expertise 2",
    -           "expertise 3",
    -           "expertise 4",
    -           "expertise 5",
    -         ],
    -         "firstName": "Tom",
    -         "institution": "some institution",
    -         "jobTitle": "some job title",
    -         "labs": Array [
    -           Object {
    -             "flatData": Object {
    -               "name": "Brighton",
    -             },
    -             "id": "cd7be4902",
    -           },
    -           Object {
    -             "flatData": Object {
    -               "name": "Liverpool",
    -             },
    -             "id": "cd7be4903",
    -           },
    -         ],
    -         "lastModifiedDate": "",
    -         "lastName": "Hardy",
    -         "onboarded": true,
    -         "orcid": "123-456-789",
    -         "orcidLastModifiedDate": null,
    -         "orcidLastSyncDate": null,
    -         "orcidWorks": Array [
    -           Object {
    -             "doi": "test-doi",
    -             "id": "123-456-789",
    -             "lastModifiedDate": "2020-10-26T15:33:18Z",
    -             "publicationDate": Object {},
    -             "title": "orcid work title",
    -             "type": "ANNOTATION",
    -           },
    -         ],
    -         "questions": Array [
    -           Object {
    -             "question": "Question 1",
    -           },
    -           Object {
    -             "question": "Question 2",
    -           },
    -         ],
    -         "reachOut": "some reach out",
    -         "researchInterests": "some research interests",
    -         "responsibilities": "some responsibilities",
    -         "role": "Grantee",
    -         "social": null,
    -         "teams": Array [
    -           Object {
    -             "id": Array [
    -               Object {
    -                 "flatData": Object {
    -                   "displayName": "Team A",
    -                   "proposal": Array [
    -                     Object {
    -                       "id": "proposalId1",
    -                     },
    -                   ],
    -                 },
    -                 "id": "team-id-0",
    -               },
    -             ],
    -             "role": "Lead PI (Core Leadership)",
    -           },
    -         ],
    -       },
            "id": "user-id-1",
    -       "institution": "some institution",
    -       "jobTitle": "some job title",
    -       "labs": Array [
    -         Object {
    -           "flatData": Object {
    -             "name": "Brighton",
    -           },
    -           "id": "cd7be4902",
    -         },
    -         Object {
    -           "flatData": Object {
    -             "name": "Liverpool",
    -           },
    -           "id": "cd7be4903",
    -         },
    -       ],
    -       "lastModified": "2020-10-26T15:33:18Z",
    -       "lastModifiedDate": "",
            "lastName": "Hardy",
    -       "onboarded": true,
    -       "orcid": "123-456-789",
    -       "orcidLastModifiedDate": null,
    -       "orcidLastSyncDate": null,
    -       "orcidWorks": Array [
    -         Object {
    -           "doi": "test-doi",
    -           "id": "123-456-789",
    -           "lastModifiedDate": "2020-10-26T15:33:18Z",
    -           "publicationDate": Object {},
    -           "title": "orcid work title",
    -           "type": "ANNOTATION",
    -         },
    -       ],
    -       "questions": Array [
    -         Object {
    -           "question": "Question 1",
    -         },
    -         Object {
    -           "question": "Question 2",
    -         },
    -       ],
    -       "reachOut": "some reach out",
    -       "researchInterests": "some research interests",
    -       "responsibilities": "some responsibilities",
    -       "role": "Grantee",
    -       "social": null,
    -       "teams": Array [
    -         Object {
    -           "id": Array [
    -             Object {
    -               "flatData": Object {
    -                 "displayName": "Team A",
    -                 "proposal": Array [
    -                   Object {
    -                     "id": "proposalId1",
    -                   },
    -                 ],
    -               },
    -               "id": "team-id-0",
    -             },
    -           ],
    -           "role": "Lead PI (Core Leadership)",
    -         },
    -       ],
    -       "version": 42,
          },
        },
      ]
```